awx/main/management/commands/run_heartbeet.py

```diff
@@ -56,6 +57,8 @@ def do_hearbeat_loop(self):
                 logger.debug('Sending heartbeat')
                 conn.notify('web_heartbeet', self.construct_payload())
                 time.sleep(settings.BROADCAST_WEBSOCKET_BEACON_FROM_WEB_RATE_SECONDS)
+                signal.signal(signal.SIGTERM, conn.notify('web_heartbeet', self.construct_payload(action='offline')))
```

Comment:
the signal only needs to be called once I think. if so we may move this to before the while True loop

Reply:
@jessicamack you're right we need both for SIGTERM and SIGINT

what I meant is that I don't think we need to put them into the while True block